ios/FluentUI.Demo/FluentUI.Demo/Demos/PreviewCardDemoController.swift

```diff
@@ -16,15 +16,19 @@ class PreviewCardDemoController: DemoTableViewController {
     }
     override func viewDidLoad() {
         super.viewDidLoad()
-        let previewCard = MSFPreviewCard(theme: nil)
-        let previewCardView = previewCard.view
-        view.addSubview(previewCardView)
-        previewCardView.translatesAutoresizingMaskIntoConstraints = false
+        let previewFlatCard = MSFPreviewCard(theme: nil, elevated: false)
```

Comment:
Maybe assign a default value to `elevated`?

Reply:
And rename it to `isElevated` to keep the standard?